src/Test/Functional/Acceptance/PatchApplierCest.php

```diff
@@ -33,7 +33,6 @@ public function testApplyingPatch(\CliTester $I)
         $I->assertContains('## Additional Info', $targetFile);
         $log = $I->grabFileContent('/var/log/cloud.log', Docker::BUILD_CONTAINER);
         $I->assertContains('INFO: Applying patch /var/www/magento/m2-hotfixes/patch.patch', $log);
-        $I->assertContains('INFO: git apply /var/www/magento/m2-hotfixes/patch.patch', $log);
```

Comment:
Maybe is better not to remove this line but change `INFO` -> `DEBUG` and add 
```
$I->uploadToContainer('files/debug_logging/.magento.env.yaml', '/.magento.env.yaml', Docker::BUILD_CONTAINER);
```
as for `AdminCredentialCest`

Reply:
I thought about this, but thought it wasn't as important because of the line directly above it which essentially tests the same thing:
` $I->assertContains('INFO: Applying patch /var/www/magento/m2-hotfixes/patch.patch', $log);`

Do you think it's important to test the "git apply" output in addition to the "applying patch" output, even if they both have the same filename?

Reply:
I'm not sure if it's important, but if Author of the test added this check he thought it was important.

Reply:
I'll add it back then. Thanks for the quick reply.